docs/design.md

```diff
@@ -967,6 +864,92 @@ spare capacity is chosen in the same datacenter and a special-case split
 is done which simply duplicates the data 1:1 and resets the range
 configuration metadata.
 
+# Raft - Consistency of Range Replicas
+
+Each range is configured to consist of three or more replicas, as specified by
+their ZoneConfig. The replicas in a range maintain their own instance of a
+distributed consensus algorithm. We use the [*Raft consensus algorithm*](https://ramcloud.stanford.edu/wiki/download/attachments/11370504/raft.pdf)
+as it is simpler to reason about and includes a reference implementation
+covering important details. Every write to replicas is logged twice.
+Once to RocksDB’s internal log and once to levedb itself as part of the
```

Comment:
"Logged twice" isn't exactly right. It's logged once, but the log is stored in RocksDB and is subject to RocksDB's write amplification (which may be as low as 1x if we are aggressive about log truncation and can purge the log before a minor compaction, or it may be higher than 2x if the log entry survives a major compaction). I'd probably just drop this bit for now and maybe reintroduce it later as a part of a discussion about disk-level performance.